Cassowary/ClLinearExpression.cs

```diff
@@ -242,7 +242,7 @@ public ClLinearExpression SetVariable(ClAbstractVariable v, double c)
         /// </summary>
         public ClLinearExpression AddVariable(ClAbstractVariable v, double c, ClAbstractVariable subject, ClTableau solver)
         {
-            ClDouble coeff = _terms[v];
+            ClDouble coeff = _terms.ContainsKey(v)?_terms[v]:null;
```

Comment:
Rather than ContainsKey followed by getting the value, use [`TryGetValue`](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.trygetvalue?view=netcore-3.1) to do both operations at once.

Reply:
Good point. Fixed